script.js

```diff
@@ -1,4 +1,300 @@
+      if (differenceValueOutput == 2) {
+        myOutputValue = "You win";
+      }
+      if (differenceValueOutput == 0) {
+        myOutputValue = "Draw...";
+      }
+    }
+    console.log("myoutput");
+    console.log(myOutputValue);
+    if (myOutputValue === "You win") {
+      countWins = countWins + 1;
+    }
+    if (myOutputValue === "You lose") {
+      countLosses = countLosses + 1;
+    }
+    return (
+      "Hi " +
+      userName +
+      "! The computer chose " +
+      computerGuessString +
+      ".<br>" +
+      "And you chose " +
+      input +
+      ".<br>" +
+      myOutputValue +
+      ".<br><br> Type 'scissors' 'paper' or 'stone' to play another round!" +
+      "<br><br>You have played " +
+      countGames +
+      " " +
+      (countGames === 1 ? "game" : "games") +
+      "<br><br>You have won " +
+      countWins +
+      " " +
+      (countWins === 1 ? "game" : "games") +
+      "<br><br>You have lost " +
+      countLosses +
+      " " +
+      (countLosses === 1 ? "game" : "games") +
+      "<br><br>Your winning percentage is " +
+      ((countWins / countGames) * 100).toFixed(2) +
+      "%" +
+      (((countWins / countGames) * 100).toFixed(2) > 50
+        ? "<br>You are doing well!"
+        : "<br>You can still improve!")
+    );
+  }
+  //korean mode
+  if (gameMode === "ready to go" && koreanMode === "korean") {
+    console.log("gameoption");
+    console.log(gameOption);
+    countGames = countGames + 1;
+    previousWinnerActual = previousWinner;
+    differenceValueOutput = difference(input);
+    var myOutputValue =
+      "Try again: you need to chose between scissors, paper and rock!";
+    if (gameOption === "reverse") {
+      if (differenceValueOutput == -1) {
+        myOutputValue = "You lose";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == 1) {
+        myOutputValue = "You win";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == 2) {
+        myOutputValue = "You lose";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == -2) {
+        myOutputValue = "You win";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == 0) {
+        myOutputValue = "Draw...";
+        previousWinner = myOutputValue;
+      }
+    } else {
+      if (differenceValueOutput == 1) {
+        myOutputValue = "You lose";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == -1) {
+        myOutputValue = "You win";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == -2) {
+        myOutputValue = "You lose";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == 2) {
+        myOutputValue = "You win";
+        previousWinner = myOutputValue;
+      }
+      if (differenceValueOutput == 0) {
+        myOutputValue = "Draw...";
+        previousWinner = myOutputValue;
+      }
```

Comment:
You repeated the same logic as the code above. Use function to abstract the logic and reuse it instead of typing the same logic. Also, remember to use Boolean operator such as " ||" ,"&&" in the future.